composer.json

```diff
@@ -13,38 +13,49 @@
         }
     ],
     "require": {
-        "php": ">=5.5",
-        "ext-json": "*",
-        "ext-simplexml": "*",
-        "guzzlehttp/guzzle": "^6.0",
-        "psr/log": "^1.0",
-        "symfony/config": "^2.1|^3.0",
-        "symfony/console": "^2.1|^3.0",
-        "symfony/stopwatch": "^2.0|^3.0",
-        "symfony/yaml": "^2.0|^3.0"
+        "php":               ">=5.6",
+        "ext-json":          "*",
+        "ext-simplexml":     "*",
+        "guzzlehttp/guzzle": ">=6.0",
+        "psr/log":           ">=1.0",
+        "symfony/config":    "*",
+        "symfony/console":   "*",
+        "symfony/stopwatch": "*",
+        "symfony/yaml":      "*"
+    },
+    "require-dev": {
+        "phpunit/phpunit":           ">=5.0",
+        "apigen/apigen":             ">=4.0",
+        "friendsofphp/php-cs-fixer": ">=2.0",
+        "pdepend/pdepend":           ">=2.0",
+        "phpmd/phpmd":               ">=2.0",
+        "sebastian/finder-facade":   ">=1.0",
+        "sebastian/phpcpd":          ">=2.0",
+        "squizlabs/php_codesniffer": ">=2.0",
+        "theseer/fdomdocument":      ">=1.0"
```

Comment:
it was moved to separated file for a reason. would be great to ask for it in advance instead of changing